sopel/modules/currency.py

```diff
@@ -1,102 +1,189 @@
+    for to in others:
+        try:
+            out_string = build_reply(bot, amount, of.upper(), to.upper(), out_string)
+        except (requests.exceptions.HTTPError, requests.exceptions.ConnectionError) as e:
+            bot.reply("Something went wrong while I was getting the exchange rate.")
+            stderr("Error in GET request: {}".format(e))
+            return NOLIMIT
+        except ValueError:
+            return NOLIMIT
+
+    bot.reply(out_string[0:-1])
+
+
+@commands('cur', 'currency', 'exchange')  # this
+@example('.cur 100 usd in btc cad eur')
+def exchange_cmd(bot, trigger):
+    if not trigger.group(2):
+        return bot.reply("No search term. An example: .cur 100 usd in btc cad eur")
+
+    match = regex.match(trigger.group(2))
+    exchange(bot, match)
+
+
+@rule(regex)
+@example('100 usd in btc cad eur')
+def exchange_re(bot, trigger):
+    if bot.config.currency.enable_regex:
+        match = regex.match(trigger)
+        exchange(bot, match)
 
 
-def display(bot, amount, of, to):
+def build_reply(bot, amount, of, to, out_string):
     if not amount:
         bot.reply("Zero is zero, no matter what country you're in.")
+
+    rate_raw = ''
     try:
-        of_rate, of_name = get_rate(of)
-        if not of_name:
-            bot.reply("Unknown currency: %s" % of)
-            return
-        to_rate, to_name = get_rate(to)
-        if not to_name:
-            bot.reply("Unknown currency: %s" % to)
-            return
-    except Exception:  # TODO: Be specific
-        bot.reply("Something went wrong while I was getting the exchange rate.")
-        return NOLIMIT
+        rate_raw = get_rate(of, to)
+        rate = float(rate_raw)
+    except ValueError:
+        bot.reply(rate_raw)
+        raise
 
-    result = amount / of_rate * to_rate
-    bot.say("{:.2f} {} ({}) = {:.2f} {} ({})".format(amount, of.upper(), of_name,
-                                             result, to.upper(), to_name))
+    result = float(rate * amount)
 
+    if to == 'BTC':
+        return out_string + ' {:.5f} {},'.format(result, to)
 
-@commands('btc', 'bitcoin')
-@example('.btc 20 EUR')
-def bitcoin(bot, trigger):
-    # if 2 args, 1st is number and 2nd is currency. If 1 arg, it's either the number or the currency.
-    to = trigger.group(4)
-    amount = trigger.group(3)
-    if not to:
-        to = trigger.group(3) or 'USD'
-        amount = 1
+    return out_string + ' {:.2f} {},'.format(result, to)
+
+
+class CurrencySection(StaticSection):
+    fixer_io_key = ValidatedAttribute('fixer_io_key', default=None)
+    enable_regex = ValidatedAttribute('enable_regex', parse=bool, default=False)
+
+
+def configure(config):
+    config.define_section('currency', CurrencySection, validate=False)
+    config.currency.configure_setting('fixer_io_key', 'API key for fixer IO. Leave blank to use exchangeratesapi.io:')
+    config.currency.configure_setting('enable_regex', 'automatically respond to regex matches:')
```

Comment:
```suggestion
    config.currency.configure_setting('enable_regex', 'Automatically respond to regex matches?')   
```

Most other settings like this phrase the configuration prompt as a question; e.g., https://github.com/sopel-irc/sopel/blob/eb23031ea7729836de6eaece61b42434a8b44fd5/sopel/modules/admin.py#L37-L40